Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
         stage('sphinx') {
           agent { label 'small' }
           steps {
+            sh 'rm -rf ./blackbox/.venv'
```

Comment:
Could use `sh 'git clean -xdff'` to align this with the other stages.

Reply:
thx! updated commit (and its description)